TypeCobol.LanguageServer.Test/LSRTests/GoToDefinition/input/GoToDefinition.tlsp

```diff
@@ -1,27 +1,107 @@
+  "didOpen": "{\"jsonrpc\":\"2.0\",\"method\":\"textDocument/didOpen\",\"params\":{\"textDocument\":{\"uri\":\"file:/C:/Users/LANOYDO/AppData/Local/Temp/2/tcbl/DVZZDLY1941980203030507940.cee\",\"languageId\":\"__lsp4j_TypeCobol\",\"version\":0,\"text\":\"       IDENTIFICATION DIVISION.\\r\\n       PROGRAM-ID. DVZS0OSM.\\r\\n       data division.\\r\\n       working-storage section.\\r\\n       COPY YFOO replacing \\u003d\\u003d:toto:\\u003d\\u003d by \\u003d\\u003dtoto\\u003d\\u003d.\\r\\n       01 Group1.\\r\\n          05 VarA pic X.\\r\\n\\r\\n       procedure division.\\r\\n           perform TOTO\\r\\n           move Group1::VarA to Group1::VarA\\r\\n           move VarA to Group1::VarA\\r\\n           goback.\\r\\n       TOTO.\\r\\n           continue.\\r\\n       END PROGRAM DVZS0OSM.\"}}}",
   "messages": [
     {
       "category": 1,
-      "message": "{\"jsonrpc\":\"2.0\",\"method\":\"textDocument/publishDiagnostics\",\"params\":{\"uri\":\"file:///C:/Users/BALLMA/AppData/Local/Temp/1/tcbl/TCOZBMX06757771808740006468.cee\",\"diagnostics\":[]}}"
+      "message": "{\"jsonrpc\":\"2.0\",\"method\":\"textDocument/publishDiagnostics\",\"params\":{\"uri\":\"file:///C:/Users/LANOYDO/AppData/Local/Temp/2/tcbl/DVZZDLY1941980203030507940.cee\",\"diagnostics\":[]}}"
     },
     {
       "category": 0,
-      "message": "{\"jsonrpc\":\"2.0\",\"id\":\"2\",\"method\":\"textDocument/definition\",\"params\":{\"textDocument\":{\"uri\":\"file:/C:/Users/BALLMA/AppData/Local/Temp/1/tcbl/TCOZBMX06757771808740006468.cee\"},\"uri\":\"file:/C:/Users/BALLMA/AppData/Local/Temp/1/tcbl/TCOZBMX06757771808740006468.cee\",\"position\":{\"line\":7,\"character\":22}}}"
+      "message": "{\"jsonrpc\":\"2.0\",\"id\":\"13\",\"method\":\"textDocument/definition\",\"params\":{\"textDocument\":{\"uri\":\"file:/C:/Users/LANOYDO/AppData/Local/Temp/2/tcbl/DVZZDLY1941980203030507940.cee\"},\"uri\":\"file:/C:/Users/LANOYDO/AppData/Local/Temp/2/tcbl/DVZZDLY1941980203030507940.cee\",\"position\":{\"line\":9,\"character\":19}}}"
     },
     {
       "category": 2,
-      "message": "{\"jsonrpc\":\"2.0\",\"id\":\"2\",\"result\":{\"uri\":\"file:/C:/Users/BALLMA/AppData/Local/Temp/1/tcbl/TCOZBMX06757771808740006468.cee\",\"range\":{\"start\":{\"line\":9,\"character\":0}}}}"
+      "message": "{\"jsonrpc\":\"2.0\",\"id\":\"13\",\"result\":{\"uri\":\"file:/C:/Users/LANOYDO/AppData/Local/Temp/2/tcbl/DVZZDLY1941980203030507940.cee\",\"range\":{\"start\":{\"line\":13,\"character\":0}}}}"
+    },
+    {
+      "category": 0,
+      "message": "{\"jsonrpc\":\"2.0\",\"id\":\"14\",\"method\":\"textDocument/hover\",\"params\":{\"textDocument\":{\"uri\":\"file:/C:/Users/LANOYDO/AppData/Local/Temp/2/tcbl/DVZZDLY1941980203030507940.cee\"},\"uri\":\"file:/C:/Users/LANOYDO/AppData/Local/Temp/2/tcbl/DVZZDLY1941980203030507940.cee\",\"position\":{\"line\":10,\"character\":18}}}"
```

Comment:
Is it possible to avoid hover requests in the test ? The test is fine but when including hover requests you also involuntarily test the hover feature so this test will be affected by #1386.

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/tree/d3a46707a09a21d2a87a1654eba1c3af820e4343